wgsl/index.bs

```diff
@@ -3808,6 +3808,115 @@ If you want `vec4<f32>` you must provide 4 float values in the constructor.
 
 Issue: (dsinclair) Write out precedence rules. Matches c and glsl rules ....
 
+Issue: (mehmetoguzderin) Refine precedence rules ....
+
+<table class='data'>
+  <thead>
+    <tr>
+      <td>Precedence
+      <td>Operators
+      <td>Description
+      <td>Associativity
+  </thead>
+  <tr>
+    <td rowspan='1' style='text-align: left'><b>1 (highest)</b>
+    <td> <b><code>(</code></b><i>,</i> <b><code>)</code></b>
```

Comment:
WGSL has no , operator

Reply:
@dj2 that's intended as separator between two tokens

Reply:
Separated with space now